reference-architecture/azure-ansible/node.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+#yum -y update
+yum -y install wget git net-tools bind-utils iptables-services bridge-utils bash-completion docker
```

Comment:
THis is recommended from the doc. https://docs.openshift.com/container-platform/3.3/install_config/install/host_preparation.html

So how do u want to handle?